src/main/java/io/confluent/connect/jdbc/sink/dialect/PostgreSqlDialect.java

```diff
@@ -97,4 +97,18 @@ public void apply(StringBuilder builder, String col) {
     return builder.toString();
   }
 
+  @Override
+  protected void formatColumnValue(StringBuilder builder, String schemaName, Map<String, String> schemaParameters, Schema.Type type, Object value) {
+    if (schemaName != null) {
+      super.formatColumnValue(builder, schemaName, schemaParameters, type, value);
+    }
```

Comment:
What about just copying only the `switch` on `schemaName` from the super method? Not great but then the rest would be correct in this method, right?